ios/Plugin/ContactsPlugin.swift

```diff
@@ -27,8 +26,8 @@ public class ContactsPlugin: CAPPlugin, CNContactPickerDelegate {
             permissionState = "prompt"
         case .restricted, .denied:
             permissionState = "denied"
-        case .authorized:
-            permissionState = "granted"
+        case .authorized, .limited:
+            permissionState = "granted_limited"
```

Comment:
```suggestion
            permissionState = "limited"
```

It's usually just called `limited`.